pkg/startup/startup.go

```diff
@@ -139,11 +139,13 @@ func Run() {
 
 		// Check if we're running on a kubeadm and/or rancher cluster. Any error other than not finding the respective
 		// config map should be serious enough that we ought to stop here and return.
-		kubeadmConfig, err = clientset.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(KubeadmConfigConfigMap,
-			metav1.GetOptions{})
+		kubeadmConfig, err = clientset.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(KubeadmConfigConfigMap, metav1.GetOptions{})
 		if err != nil {
 			if errors.IsNotFound(err) {
 				kubeadmConfig = nil
+			} else if errors.IsUnauthorized(err) {
```

Comment:
this looks like a reasonable inference and an improvement to spamming the log with errors that are not relevant.